sfcasts/ep1/computed.md

```diff
@@ -1,228 +1,192 @@
-# Computed
+# Computed Properties
 
 Our collapsing sidebar works, but *yikes*, it looks terrible! We *need* to fix this!
-What we need to do here is hide all of these texts and links entirely when the sidebar
-is in its collapsed state. 
+Let's hide the text and links entirely when the sidebar
+is in its collapsed state.
 
 ## v-if directive
 
-Look at `sidebar.vue` in the template, we want to hide the `<h5>`, the `<ul>` and the
-`<hr />`, basically everything except for the button, when we're in a collapsed
-state.
+Look at `sidebar.vue`. In the template, we want to hide the `<h5>`, the `<ul>` and the
+`<hr />`: basically everything except for the button.
 
 In `Vue`, there are two ways to do this and both of them come in the form of
-*directives*. At the end of this chapter we will have seen almost all of the important
-directives in `Vue`. You will be a Directive Master!
+*directives*.
 
 To make this work, I'm going to add a new `<div>` to wrap all the elements that we
-need hide and *then* apply a directive to it so that it will hide or show this element
-and all of its contents.
+need hide.
 
-The first way of doing this is via the `v-if` directive. We can write `v-if=""`, and
-inside of it, use a JavaScript expression. We'll say `!collapsed` so as to show this
-if we are *not* in the collapsed state. If we go over to our browser we don't even
-need to refresh because we have the hot module replacement on... and it works!
+The first directive is `v-if`. We can write `v-if=""` and,
+inside, use a JavaScript expression. We'll say `!collapsed`. So: show this if we
```

Comment:
```diff
- inside of it
+ inside
```

Just trying to use less words

```diff
- We'll say `!collapsed` so as to show 
+ We'll say `!collapsed`. So: show this if we
```

I made this change when I was imagining myself reading this. It sounded good as 2 statements. Also, "so as to show" is a bit formal sounding - it sounds like something that would be written in a book, but not said (if that make sense).

Reply:
👍